Update configure-sql-server-azure-monitor-sap-solutions.md (#104185)
Azure/azure-content: articles/virtual-machines/workloads/sap/configure-sql-server-azure-monitor-sap-solutions.md

ALTER ROLE [db_denydatawriter] ADD MEMBER [AMS]
GO
```
## Prerequisites to enable secure communication

> To enable [TLS 1.2 or higher](https://learn.microsoft.com/en-us/azure/virtual-machines/workloads/sap/enable-tls-azure-monitor-sap-solutions), follow the steps [mentioned here](https://learn.microsoft.com/en-us/sql/database-engine/configure-windows/configure-sql-server-encryption?view=sql-server-ver15)

Reviewer: ```suggestion
To enable [TLS 1.2 or higher](enable-tls-azure-monitor-sap-solutions.md), follow the steps [mentioned here](/sql/database-engine/configure-windows/configure-sql-server-encryption?view=sql-server-ver15&preserve-view=true).
```

1. On the **Add provider** page, enter all required information:
    1. For **Type**, select **Microsoft SQL Server**.
    1. For **Name**, enter a name for the provider.

>     1. *Optional* Select **Enable secure communication**, choose a certificate type from drop down

Reviewer: ```suggestion
    1. *Optional* Select **Enable secure communication** and choose a certificate type from the dropdown list.
```
